versioned_docs/version-2.0/admin-manual/config/fe-config.md

```diff
@@ -1497,6 +1497,12 @@ Default：1 * 3600  (1 hour)
 
 Load label cleaner will run every *label_clean_interval_second* to clean the outdated jobs.
 
+#### `label_regex_length`
+
+Default: 128 (character)
+
+The maximum character length of the load label, the default is 128 characters.
```

Comment:
"the default value is 128 characters" should have value 

and btw , it is character or characters?  “Default: 128 (character)” is singular ， the default is 128 characters is plural

Reply:
change to 128(characters)